class.jetpack.php

```diff
@@ -4458,7 +4458,7 @@ public function get_remote_query_timeout_limit() {
 	 * verifies it worked properly.
 	 *
 	 * @since 2.6
-	 * @return true or Jetpack_Error
+	 * @return json object returned by server
```

Comment:
I think this should be `@return string|Jetpack_Error A JSON object on success or Jetpack_Error on failure`.